experimental/add_to_app/README.md

```diff
@@ -71,6 +83,50 @@ These apps showcase a relatively straightforward integration of
 If you are new to Flutter's add-to-app APIs, these projects are a great place
 to begin learning how to use them.
 
+### `android_using_plugin` and `ios_plugin`
+
+These apps are similar to `android_fullscreen` and `ios_fullscreen`, with the
+following differences:
+
+* Rather than importing `flutter_module`, they import
+  `flutter_module_using_plugin`.
+* They include the native code (Kotlin or Swift) required to initialize plugins
```

Comment:
@matthew-carroll keep an eye out for this one (since we might not need this very soon)